packages/aztec.js/src/proof/bilateralSwap/helpers.js

```diff
@@ -0,0 +1,170 @@
+const BN = require('bn.js');
+const crypto = require('crypto');
+
+
+const bn128 = require('../../bn128');
+const secp256k1 = require('../../secp256k1');
+const notesConstruct = require('../../note');
+
+
+const { groupReduction } = bn128;
+
+
+const helpers = {};
+
+helpers.makeTestNotes = (makerNoteValues, takerNoteValues) => {
+    const noteValues = makerNoteValues.concat(takerNoteValues);
+    const numNotes = noteValues.length;
+
+    let i;
+    const publicKeys = [];
+    for (i = 0; i < numNotes; i += 1) {
+        const { publicKey } = secp256k1.accountFromPrivateKey(crypto.randomBytes(32));
+        publicKeys.push(publicKey);
+    }
+
+    const testNotes = publicKeys.map((publicKey, j) => {
+        return notesConstruct.create(publicKey, noteValues[j]);
+    });
+
+    return testNotes;
+};
+
+helpers.checkNumberNotes = (notes, numberExpected) => {
+    if (notes.length !== numberExpected) {
+        throw new Error('Incorrect number of notes');
+    }
+};
+
+helpers.makeNoteArray = (notes) => {
+    const makerNotes = Object.values(notes.makerNotes);
+    const takerNotes = Object.values(notes.takerNotes);
+    const noteArray = [makerNotes[0], makerNotes[1], takerNotes[0], takerNotes[1]];
+    return noteArray;
+};
+
+helpers.makeIncorrectArray = (notes) => {
```

Comment:
I can't see this being used anywhere, should be deleted

Reply:
Done